clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -2707,6 +2707,98 @@ def CIR_CallOp : CIR_CallOpBase<"call", [NoRegionArguments]> {
   ];
 }
 
+def CIR_TryCallOp : CIR_CallOpBase<"try_call",[
```

Comment:
This should probably have a verifier. The two destinations need to be valid blocks in the same region as the caller, and the landing pad destination needs to be a landing pad (though I'm not sure exactly how to verify that and it probably can't be done yet).

Reply:
OK. This can wait until more pieces are implemented.